core/src/main/java/net/opentsdb/query/anomaly/BaseAnomalyNode.java

```diff
@@ -113,8 +143,7 @@ public BaseAnomalyNode(final QueryNodeFactory factory,
     if (!Strings.isNullOrEmpty(config.getTrainingInterval())) {
```

Comment:
I think we need more logic to get the actual interval we need to give the data source, but that can come later I guess.

Reply:
Yeah, for now it's user provided.